geomesa-spark/geomesa-spark-jts/src/main/scala/org/apache/spark/sql/jts/AbstractGeometryUDT.scala

```diff
@@ -23,21 +20,11 @@ import scala.reflect._
  */
 abstract class AbstractGeometryUDT[T >: Null <: Geometry: ClassTag](override val simpleString: String)
   extends UserDefinedType[T] {
+  protected def gf: GeometryFactory = AbstractGeometryUDT.gf
```

Comment:
Did you test the PySpark support by chance?  

I wonder what we need to do in order to check that out...

Reply:
hmm, no i didn't... i would hope it just works since everything is essentially the same, but we should verify. i do not actually know how to do that though :/